rust/decentraland-godot-lib/src/godot_classes/dcl_scene_node.rs

```diff
@@ -0,0 +1,42 @@
+use godot::engine::Node3D;
+use godot::prelude::*;
+
+#[derive(GodotClass)]
+#[class(init, base=Node3D)]
+pub struct DclSceneNode {
+    scene_id: u32,
+
+    is_global: bool,
```

Comment:
This is a property of the scene, not of the entity. Could you move this to the SceneRoot and then go until that?

Reply:
misunderstood, forget it